ui/lib/core/addon/components/masked-input.js

```diff
@@ -52,7 +52,9 @@ export default Component.extend({
       this.onChange(value);
     },
     handleKeyUp(name, value) {
-      this.onKeyUp(name, value);
+      if (this.onKeyUp) {
+        this.onKeyUp(name, value);
+      }
```

Comment:
I came across an error in the console while testing where `onKeyUp` was undefined so I added a check.